doc/build-unix.md

```diff
@@ -13,7 +13,7 @@ Run the following commands to install required packages:
 
 ##### Debian/Ubuntu:
 ```bash
-$ sudo apt-get install curl build-essential libtool autotools-dev automake pkg-config python3 bsdmainutils
+$ sudo apt-get install curl build-essential libtool autotools-dev automake pkg-config python3 bsdmainutils byacc
```

Comment:
may be better to use `bison` instead `byacc`, `byacc` can't compile on Kubuntu 21.10, strange.
Just tried `byacc 20140715-1build1` (failed) vs `bison 2:3.7.6+dfsg-1build1` (worked)

Reply:
Makes sense, `bison` is required for linux cross compilation too https://github.com/dashpay/dash/blob/develop/depends/README.md?plain=1#L55